sources/main.cpp

```diff
@@ -20,11 +20,73 @@
 #include <DYNIoDico.h>
 #include <DYNMPIContext.h>
 #include <boost/filesystem.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/property_tree/ptree.hpp>
 #include <chrono>
 #include <cstdlib>
 #include <sstream>
 
-static std::string
+struct SimulationParams {
```

Comment:
We should'nt have a struct defined in the main as it makes the reading difficult

Reply:
Indeed, though I was wondering where should it be defined. Do you have any suggestion?